xds/src/test/java/io/grpc/xds/CdsLoadBalancer2Test.java

```diff
@@ -783,7 +785,7 @@ public void shutdown() {
     }
   }
 
-  private final class FakeXdsClient extends XdsClient {
+  private static class FakeXdsClient extends XdsClient {
```

Comment:
Why no longer final?

Reply:
How is that related? It can be both static and final. I guess it doesn't matter because this is a test, but still looked strange.